README.md

```diff
@@ -19,69 +21,81 @@ You can also permanently install Kitten Scientists with a userscript manager.
 
-Hunts when catpower is at 98% capacity.
+Trading also has a *limited* trading mode. This mode determines how much production time is needed to make the trade's input resources versus the time to make the trade's average output resources to determine if a trade is profitable. The resources gained and spent trading and hunting are also factored in, making this mode self-limiting.
 
-### Praising
+### Religion
 
-Praises when faith is at 98% capacity.
+> This section has a configurable threshold for when the automation should be activated. Click the **trigger** label to the right of the **Religion** label to set your desired threshold. The default value is `0` (activate as soon as possible).
 
-### Misc
-KS can be set to:
+In this section, you can select which buildings and techs from the **Religion** page you want to have purchased automatically.
 
-Automatically buy cryptocurrency at low prices and sells near the peak.
+### Time
 
-Automatically hold festivals when income is sufficiently high to support them.
+> This section has a configurable threshold for when the automation should be activated. Click the **trigger** label to the right of the **Time** label to set your desired threshold. The default value is `0` (activate as soon as possible).
 
-Automatically build embassies.
+In this section, you can select which buildings from the **Time** page you want to have built automatically.
 
-Autofeed necrocorns to leviathans.
+### Options
 
-### Game Log
+- **Observe Astro Events**: Automatically observe astronomical events as they happen.
+- **Hold Festivals**: Automatically hold festivals as soon as the previous one ends and enough resources are available.
+- **Auto Praise**: Automatically praises the sun as your faith approaches the resource limit.
+- **Force Ships to 243**: Ensures that 243 ships are constructed as soon as possible, ignoring other resource constraints. This is useful because after 243 ships, trades are guaranteed to be successful 100% of the time.
+- **Feed Leviathans**: Automatically feed the leviathans race necrocorns to ensure they stay longer for possible trading.
+- **Hunt**: Automatically send your kittens hunting as you approach the catpower limit. This automation has a configurable trigger. The default value is `0.98`, which means the kittens will hunt when your catpower is at 98% of your catpower limit.
+- **Trade Blackcoin**: Automatically trade blackcoin with the leviathans at low prices and sells near the peak.
+- **Build Embassies (Beta)**: _pending documentation_
+- **Explore (deprecated)**: _relates to a feature no longer present in current versions of Kittens Game_
 
-Automatically observes astronomical events.
+### Filters
 
-All KS log output can be filtered.
+In this section, you can set which types of messages you want to see in the game log.
```

Comment:
> In this section, you can

I'm not sure how, but might want to remove this redundancy in the document. That starting line is repeated in almost every section.

Reply:
Yeah, I didn't like that either, but didn't know how to make it better at the time. I'll try to come up with something better.